backend/src/main/java/ai/verta/modeldb/entities/ExperimentRunEntity.java

```diff
@@ -180,7 +180,7 @@ public ExperimentRunEntity(ExperimentRun experimentRun) throws InvalidProtocolBu
       mappedBy = "experimentRunEntity",
       cascade = CascadeType.ALL)
   @LazyCollection(LazyCollectionOption.FALSE)
-  private List<VersioningModeldbEntityMapping> versioningModeldbEntityMappings = new ArrayList<>();
+  private List<VersioningModeldbEntityMapping> versioned_inputs = new ArrayList<>();
```

Comment:
Why the snake case? `versionedInputs`

Reply:
we have used other fields as snack cases also. the reason is we have used the same name as the proto name so when we called FindExperimentRun with KeyValueQuery we just pass `hyperparameter.loss` & `versioned_inputs.repository_id` as a key and used that key directly in the Criteria query.